schemas/creativeWork.schema.tpl.json

```diff
@@ -0,0 +1,141 @@
+{
+  "properties": {
+    "about": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add links to the datasets, models or software contained within or described by the creative work.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/DatasetVersion",
+        "https://openminds.ebrains.eu/core/ModelVersion",
+        "https://openminds.ebrains.eu/core/SoftwareVersion"
+      ]
+    },
+    "abstract": {
+      "type": "string",
+      "_instruction": "Enter the abstract of the creative work."
+    },
+    "custodian": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Enter the person or people responsible for the creative work (e.g. the corresponding author).",
+      "_linkedCategories": [
+        "legalPerson"
+      ],
+      "_originalName": "accountablePerson"
+    },
+    "shortName": {
+      "type": "string",
+      "maxLength": 30,
+      "_instruction": "Enter a short name (alias) for this creative work (max. 30 characters; no space)."
+    },
+    "author": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add one or several authors (person or organization) that contributed to the production and publication of this creative work.",
+      "_linkedCategories": [
+        "legalPerson"
+      ]
+    },
+    "citation": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add citations or references from this work to another creative work",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/publications/Book",
+        "https://openminds.ebrains.eu/publications/Chapter",
+        "https://openminds.ebrains.eu/publications/ScholarlyArticle",
+        "https://openminds.ebrains.eu/publications/LivePaper"
+      ]
+    },
+    "copyright": {
+      "_instruction": "Add the copyright information of this research product version.",
+      "_embeddedTypes": [
+        "https://openminds.ebrains.eu/core/Copyright"
+      ],
+      "_originalName": ["copyrightHolder", "copyrightYear"]
+    },
+    "creationDate": {
```

Comment:
is this really a point of interest? I would keep only the release date (but we can keep it of course)

Reply:
about the "citation" field, it is optional.

about the "creationDate" field, we could remove it. It's often helpful to have a date field associated with unreleased items, however, which is why I left it in (from schema.org)